docs/content/envoy-authorization.md

```diff
@@ -259,63 +270,62 @@ spec:
             runAsNonRoot: false
             runAsUser: 0
       containers:
-        - name: app
-          image: openpolicyagent/demo-test-server:v1
-          ports:
-            - containerPort: 8080
-        - name: envoy
-          image: envoyproxy/envoy:v1.14.4
-          securityContext:
-            runAsUser: 1111
-          volumeMounts:
-          - readOnly: true
-            mountPath: /config
-            name: proxy-config
-          args:
-          - "envoy"
-          - "--config-path"
-          - "/config/envoy.yaml"
-        - name: opa
-          # Note: openpolicyagent/opa:latest-envoy is created by retagging
-          # the latest released image of OPA-Envoy.
-          image: openpolicyagent/opa:{{< current_opa_envoy_docker_version >}}
-          securityContext:
-            runAsUser: 1111
-          volumeMounts:
-          - readOnly: true
-            mountPath: /policy
-            name: opa-policy
-          args:
-          - "run"
-          - "--server"
-          - "--addr=localhost:8181"
-          - "--diagnostic-addr=0.0.0.0:8282"
-          - "--set=plugins.envoy_ext_authz_grpc.addr=:9191"
-          - "--set=plugins.envoy_ext_authz_grpc.query=data.envoy.authz.allow"
-          - "--set=decision_logs.console=true"
-          - "--ignore=.*"
-          - "/policy/policy.rego"
-          livenessProbe:
-            httpGet:
-              path: /health?plugins
-              scheme: HTTP
-              port: 8282
-            initialDelaySeconds: 5
-            periodSeconds: 5
-          readinessProbe:
-            httpGet:
-              path: /health?plugins
-              scheme: HTTP
-              port: 8282
-            initialDelaySeconds: 5
-            periodSeconds: 5
+      - name: app
+        image: openpolicyagent/demo-test-server:v1
+        ports:
+        - containerPort: 8080
+      - name: envoy
+        image: envoyproxy/envoy:v1.17.0
+        volumeMounts:
+        - readOnly: true
+          mountPath: /config
+          name: proxy-config
+        args:
+        - "envoy"
+        - "--config-path"
+        - "/config/envoy.yaml"
+        env:
+        - name: ENVOY_UID
+          value: "1111"
```

Comment:
I've reduced the indentation a bit, so this doesn't stand out as much as it should -- this is the piece that fixes

```
chown: changing ownership of '/dev/stdout': Operation not permitted                                                                                                       
chown: changing ownership of '/dev/stderr': Operation not permitted 
```